apps/opik-backend/src/main/java/com/comet/opik/api/Span.java

```diff
@@ -58,13 +58,15 @@ public record Span(
         @JsonView({
                 Span.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY, description = "Duration in milliseconds as a decimal number to support sub-millisecond precision") Double duration){
 
+    @Builder(toBuilder = true)
     public record SpanPage(
             @JsonView(Span.View.Public.class) int page,
             @JsonView(Span.View.Public.class) int size,
             @JsonView(Span.View.Public.class) long total,
-            @JsonView(Span.View.Public.class) List<Span> content) implements com.comet.opik.api.Page<Span> {
+            @JsonView(Span.View.Public.class) List<Span> content,
+            @JsonView(Trace.View.Public.class) List<String> sortableBy) implements com.comet.opik.api.Page<Span> {
```

Comment:
Just noticed this minor thing:
```suggestion
            @JsonView(Span.View.Public.class) List<String> sortableBy) implements com.comet.opik.api.Page<Span> {
```

Reply:
Good catch thank you!